codebases/index.md

```diff
@@ -0,0 +1,37 @@
+
+# Where we work: our codebases
+
+The Foundation for Public Code helps to effectively scale codebases. We do this by supporting ambitious communities to build shared governance practices and smooth collaboration around their codebases.
+
+Codebases are organized by [codebase maturity](https://about.publiccode.net/activities/codebase-stewardship/lifecycle.html).
+
+## Assessment
+
+In this phase, codebase communities and the Foundation for Public Code decide if:
+
+* the codebase is a good fit for Foundation stewardship
+* Foundation stewardship will meet the community's goals and ambitions
+
+These codebases are currently in public assessment:
+
+* [Signalen](https://github.com/Amsterdam/signals): receiving, automatically classifying and prioritising residents’ nuisance reports
+
+## Incubation
+
+Codebases in incubation are working to comply with the [Standard for Public Code](https://standard.publiccode.net/).
+
+These codebases are currently in incubation:
+
+* [OpenZaak](https://github.com/open-zaak/open-zaak): backend component to help municipalities manage and share data internally
+
+## Mature
+
+Mature codebases and communities fully meet the Standard for Public Code.
+
+There are currently no mature codebases under Foundation stewardship.
+
+## Attic
+
+Codebases and communitities that were once Standard compliant but aren't any more are in the attic.
+
+There are currently no codebases in our attic.
```

Comment:
I'd say this is not where we do our work, and also is not relevant yet. i don't see what this section adds.

Reply:
I had included so that the lifecycle model was complete (for educational purposes). I've cut it now because:
- you're right that we don't anticipate anyone caring about this for a long time
- there are more valuable things we'd like people to pay attention to

Reply:
Update: we're adding this section back in in #111. @angelplasma reckons that nothing being in the attic is important to show as well - that we can be proud that nothing once Standard compliant has fallen out of usage, suffered a community collapse, or changed its values signficantly.